js-api-spec/legacy/importer.node.test.ts

```diff
@@ -35,7 +35,29 @@ it('an empty object means an empty file', () =>
 
 describe('import precedence:', () => {
   describe('in sandbox dir', () => {
-    it('relative file is #1', () =>
+    it('packageImporter:node is #1', () =>
```

Comment:
This isn't relevant to the import precedence that this `describe()` is testing. These tests verify what happens when there are multiple possible interpretations of the same load. Because the package importer is always absolute (since it has the `pkg:` scheme), it's never ambiguous with relative, load path, or working-directory loads.

Reply:
That makes sense. I've removed this test here. I added this as the spec specifies that Package Importers are resolved first for the legacy API. Do we need to test that, or perhaps, is there really a way to test that, as there can't be conflicts?